tools/data_integrity/fix_candidate_age.php

```diff
@@ -19,7 +19,16 @@
  */
 require_once __DIR__ . '/../generic_includes.php';
 
-$instruments = \NDB_BVL_Instrument::getInstrumentsList();
+$instruments = \NDB_BVL_Instrument::getInstrumentsList(
+    new \LORIS\LorisInstance(
+        $DB,
+        $config,
+        [
+            __DIR__ . "/../../project/modules",
+            __DIR__ . "/../../modules/",
+        ],
+    )
+);
```

Comment:
@laemtl could you define the lorisinstance in the generic_includes.php so all tools can use it ?

Reply:
@ridz1208 Sure!

Reply:
@ridz1208 Done.